SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKDefaultLaunchConfigGenerator.cpp

```diff
@@ -193,17 +171,90 @@ bool GenerateLaunchConfig(const FString& LaunchConfigPath, const FSpatialLaunchC
 			return false;
 		}
 
+		if (bGenerateCloudConfig)
+		{
+			return ConvertToClassicConfig(LaunchConfigPath, InLaunchConfigDescription);
```

Comment:
Will we be able to remove this when we switch to SVP?

Reply:
Yeah that's correct. Will make a ticket.